public/components/FeatureAnywhereContextMenu/CreateAnomalyDetector/AssociateExisting/containers/AssociateExisting.tsx

```diff
@@ -0,0 +1,282 @@
+      ),
+    [
+      associateExistingProps.selectedDetector,
+      existingDetectorsAvailableToAssociate,
+    ]
+  );
+  const options = useMemo(() => {
+    if (!existingDetectorsAvailableToAssociate) {
+      return [];
+    }
+
+    return existingDetectorsAvailableToAssociate.map((detector) => ({
+      label: detector.name,
+    }));
+  }, [existingDetectorsAvailableToAssociate]);
+
+  return (
+    <div className="associate-existing">
+      <EuiText size="xs">
+        <p>
+          View existing anomaly detectors across your system and add the
+          detector(s) to a dashboard and visualization.{' '}
+          <a href={`${BASE_DOCS_LINK}/ad`} target="_blank">
+            Learn more <EuiIcon type="popout" />
+          </a>
+        </p>
+      </EuiText>
+      <EuiSpacer size="l" />
+      <EuiTitle size="s">
+        <h3>Select detector to associate</h3>
+      </EuiTitle>
+      <EuiSpacer size="m" />
+      <EuiText size="xs" color="subdued" style={{ paddingBottom: '3px' }}>
+        Eligible detectors don't include high-cardinality detectors.
+      </EuiText>
+      {!existingDetectorsAvailableToAssociate && <EuiLoadingSpinner size="l" />}
+      {existingDetectorsAvailableToAssociate && (
+        <EuiComboBox
+          isLoading={isLoading}
+          id="associate-existing__select"
+          options={options}
+          selectedOptions={selectedOptions}
+          onChange={(selectedOptions) => {
+            let detector = {} as DetectorListItem | undefined;
+
+            if (selectedOptions && selectedOptions.length) {
+              const match = existingDetectorsAvailableToAssociate.find(
+                (detector) => detector.name === selectedOptions[0].label
+              );
+              detector = match;
+            }
+            associateExistingProps.setSelectedDetector(detector);
+          }}
+          aria-label="Select an anomaly detector to associate"
+          isClearable
+          singleSelection
+          placeholder="Search for an anomaly detector"
+        />
+      )}
+      <EuiSpacer size="xl" />
+      {detector && (
+        <>
+          <EuiFlexGroup justifyContent="spaceBetween" alignItems="flexStart">
+            <EuiFlexItem>
+              <EuiText>
+                <h4>{detector.name}</h4>
+              </EuiText>
+              <EuiSpacer size="s" />
+              <EuiHealth color={stateToColorMap.get(detector.curState)}>
+                {renderTime(detector.enabledTime)}
+              </EuiHealth>
+            </EuiFlexItem>
+            <EuiFlexItem grow={false}>
+              <EuiLink href={`${PLUGIN_NAME}#/detectors/${detector.id}`}>
+                View detector page
+              </EuiLink>
+            </EuiFlexItem>
+          </EuiFlexGroup>
+          <EuiHorizontalRule margin="m" />
+          <ul className="associate-existing__detector-details">
+            {[
+              ['Indexes', (detector) => detector.indices],
```

Comment:
Should this just be `Index`? And should we just pull the first index from `detector.indices`? Right now we only populate first index.

Reply:
I assumed we have it as indices here because we allow index patterns. I will change to pull from the first index as we do throughout the frontend though and then I guess it would be correct to just write `Index`

Reply:
From existing frontend, it looks we're already inconsistent. `Indices` on detector list page, and 'Data source index' on detector configuration page. So no hard opinion one way or the other really. But if going with multiple, we should use `Indices` instead of `Indexes` for consistency

Reply:
changed to indices here